backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConstants.java

```diff
@@ -0,0 +1,21 @@
+package gov.cdc.usds.simplereport.api.converter;
+
+public class FhirConstants {
+  private FhirConstants() {
+    throw new IllegalStateException("Utility class");
+  }
+
+  public static final String NULL_CODE_SYSTEM =
```

Comment:
I recommend adding `_URL` to the name to keep it consistent with the others

Reply:
So the reason why these don't have `_URL` at the end is because the are used as the [system](https://github.com/CDCgov/prime-simplereport/pull/4842/files/8619417398fbd503126c58189a0ca4da12d0d032#diff-0f13a6fdf468bcfcc079f96410fbd7204be5fded4c21df796d259163c3ea62c5R200) and the `_URL` are added to the [URL](https://github.com/CDCgov/prime-simplereport/pull/4842/files/8619417398fbd503126c58189a0ca4da12d0d032#diff-0f13a6fdf468bcfcc079f96410fbd7204be5fded4c21df796d259163c3ea62c5R194), but if that's confusing I can totally change it to describe what the contents of that variable is like recommended.